### PR TITLE
Deploy Atlantis

### PR DESCRIPTION
**Pull request by upodroid**
I'm deploying Atlantis to enable Terraform PR automation. Details: https://www.runatlantis.io

### CODE REVIEW

**Comment by upodroid** [comment]
/cc @ameukam @BenTheElder 

This is ready to be merged. Atlantis is up and running, but I'll add the privileges for GCP/AWS in a separate PR.

1. Atlantis will only accept commands from members of @kubernetes/sig-k8s-infra 
2. https://atlantis.k8s.io is accessible for any member of @kubernetes GitHub org except the /events path.
3. Atlantis is configured to verify webhook as described here https://docs.github.com/en/webhooks/using-webhooks/validating-webhook-deliveries hence anyone can access https://atlantis.k8s.io/events

Have a look https://github.com/kubernetes/k8s.io/pull/7522 for an example.

**Comment by ameukam** [comment]
/lgtm
/hold

For Ben's review

**Review by cpanato — APPROVED** [review]
look cool!

/lgtm

File: kubernetes/gke-utility/atlantis/atlantis.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+gh-user: k8s-infra-ci-robot
+gh-org: kubernetes
```
[the remark this answers is not in the document]
Comment (upodroid) [reply kubernetes/gke-utility/atlantis/atlantis.yaml:2]: Yes, the bot uses it to look up team membership.

**Pull request merged**